doc/modules/ROOT/pages/index.adoc

```diff
@@ -2,121 +2,112 @@
 :templates-url: templates:ROOT:page$/
 :plugin-gh-address: https://github.com/hazelcast/hazelcast-hibernate
 
-= Jcache SPI for using Hazelcast as L2C for Hibernate
+= Getting Started with Hibernate Second-Level Cache and Hazelcast with JCache
 
-Configuration deatils to use JCache SPI for Hibernate Second Level Cache backed by Hazelcast IMDG.
+This guide will show you how to use Hazelcast IMDG as Hibernate's Second-Level Cache by leveraging Hibernate's https://www.javadoc.io/doc/javax.cache/cache-api/1.1.1[JCache (JSR 107)] support.
 
-You can find the code samples for entitiy cache, collection cache, and query cache use cases {github-address}[here.]
+You will have a chance to see how to integrate Hazelcast with Hibernate without using the `hazelcast-hibernate` integration.
 
 == What You’ll Learn
 
-You will be able to use Hibernate Second Level Cache using JCache SPI rather than `Hazelcast-hibernate` plugin.
-
-This is an alternative approach of using {plugin-gh-address}[Hazelcast-hibernate] plugin to use Hazelcast as the caching provider for Hibernate. Note that the local region caches supported by the plugin are not available here. Unlike plugin - which stores query results locally, this approach supports query results to be stored in distributed map.
-
-=== About Jcache
-
-https://www.javadoc.io/doc/javax.cache/cache-api/1.1.1[JCache (JSR 107)] is a standard caching API for Java.
-It provides an API for applications to be able to create and work with in-memory cache of objects. Hibernate has
-started to support Jcache implementations for L2C with version 5.2. Using only a CachingProvider implementation,
-L2C can be used for SessionFactory objects in Hibernate.
+- How to use Hazelcast's distributed maps as entity/collection/query caches
 
 == Prerequisites
 
-Only the configuration details are explained here hence no prerequisites exists. However, in order to give {github-address}[code samples] a try, make sure you have:
+- JDK 8+
+- Apache Maven 3.2+
 
-include::{templates-url}/microservices/prerequisites.adoc[]
+== Required Dependencies
 
-== Setting Dependencies
+First, we need to add required dependencies.
 
-In `pom.xml`, include `hibernate-jcache` dependency having the same version with `hibernate-core` in your project:
+Let's include `hibernate-jcache` and `hibernate-core` dependencies. Keep in mind that both should have the same version:
 
 [source, xml]
 ----
 <dependency>
     <groupId>org.hibernate</groupId>
     <artifactId>hibernate-jcache</artifactId>
-    <version>${version.hibernate-core}</version>
+    <version>${hibernate-core.version}</version>
```

Comment:
May be make this coherent with pom.xml. This way one can conclude the same version must be used.
```suggestion
    <version>${hibernate.version}</version>
```